docs/user/quick-start.rst

```diff
@@ -1,19 +1,233 @@
+
+   i. Env Variables: PRESTO_SERVER=<YOUR_PATH_TO_PRESTO_SERVER>;DATA_DIR=/Users/<YOUR_USER_NAME>/Desktop;WORKER_COUNT=0
+
+   ii. VM Options: -ea -Xmx2G -XX:+ExitOnOutOfMemoryError -Duser.timezone=America/Bahia_Banderas -Dhive.security=legacy
+
+   iii. Main class: com.facebook.presto.hive.HiveExternalWorkerQueryRunner
+
+   
+NOTE:
+   HiveExternalWorkerQueryRunner will basically launch a testing presto service using local file system.
+   WORKER_COUNT is the number of workers to be launched along with the coordinator. In this case we put 0 as we want to externally launch our own CPP worker.
+   Note discovery URI. Something like http://127.0.0.1:54557. Use the last discovery URI in the InteliJ logs
+   
+Upon running this you should see the Presto service log printing in the console. 
+
+3. Update BDTK worker configuration
+The configuration structrue is stricly the same as Presto-java. And you can put the etc directory anywhere your like. 
+::
+
+   $ mkdir ${path-to-presto}/presto-native-execution/etc
+   $ cd etc
+   $ vim config.properties
+
+Add the basic configuration for a presto worker. Use discovery URI from the logs above and update the config.properties. 
+The config.properties file should be like: 
+::
+
+   task.concurrent-lifespans-per-task=32
+   http-server.http.port=7071
+   task.max-drivers-per-task=96
+   discovery.uri=http://127.0.0.1:54557
+   system-memory-gb=64
+   presto.version=testversion
+
+And then you need modify the node.properties
+::
+
+   $ vim node.properties
+
+The node.properties should be like:
+::
+
+   node.id=3ae8d81c-97b8-42c4-9e49-3524cfbe5d8b
+   node.ip=127.0.0.1
+   node.environment=testing
+   node.location=test-location
+
+Then you need to modify the configuration for a catalog.
+::
+
+   $ mkdir catalog
+   $ vim hive.properties
+
+Note: You don't have to configure a real hive catalog.
+In the HiveExternalWorkerQueryRunner it'll create a pseudo hive metastore for you. 
+
+The hive.properties should be like:
+
+::
+
+   connector.name=hive
+
+4. Launch BDTK worker
+   
+Go to YOUR_PATH_TO_PRESTO_SERVER: 
+::
+
+   cd ${path-to-presto}/presto-native-execution/_build/release/presto_cpp/main/
+   # launch the worker
+   ./presto_server --v=1 --logtostderr=1 --etc_dir=${path-to-your-etc-directory}
+
+When you see "Announcement succeeded: 202" printed to the console, the BDTK worker has successfully connected to the coordinator. 
+
+5. Test the queries
+You can sent out queries using your existing presto-cli our go to the presto-cli module you just compiled.
+::
+
+   $ cd ${path-to-presto}/presto-cli/target
+   $ ./presto-cli-${PRESTO_VERSION}-SNAPSHOT-executable.jar --catalog hive --schema tpch
+
+By doing this you can launch an interactive SQL command.
+Try Some queries with BDTK!
+
+
+
+Run a DEMO using HDFS
```

Comment:
Hmm, we can create one section called ```run with Prestodb```. Then put ```Run a DEMO using HDFS``` and ```Integrate BDTK with Presto``` as sub-sections to that one. And mentioning it's alternative way to do this.